platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/location/LocationComponentOptions.java

```diff
@@ -1973,6 +1999,16 @@ public LocationComponentOptions.Builder pulseInterpolator(Interpolator pulseInte
       return this;
     }
 
+    /**
+     * Enable or disable the concurrent camera animations during navigation feature
+     *
+     * @return whether concurrent camera animations are enabled or disabled
+     */
+    public LocationComponentOptions.Builder concurrentCameraAnimationsEnabled(Boolean concurrentCameraAnimationsEnabled) {
```

Comment:
Enabling concurrent camera animations is not only relevant for the LocationComponent, is it? So it should not be a setting that exists only here.

Reply:
Currently that's the only use case I can think of. If you want to have multiple animations at the same time from the application side then you could just bundle the animations into a single `setCamera` call.

If you still think that concurrent camera animations should be universal then I'd also have to change the logic around `cancelTransitions` to not check for the current camera mode.

Reply:
> If you want to have multiple animations at the same time from the application side then you could just bundle the animations into a single `setCamera` call.

Why is this not possible in the location component? 

What if someone wants to implement their own location component?

Reply:
I think I understand. Would it make more sense to add a boolean to the java `Transform` class and expose the concurrent animations flag from the `MapView` class instead?